Reflections/R0 - April 9.md

```diff
@@ -0,0 +1,18 @@
+## 1. What have you accomplished this week? (Please list each accomplishment, and explain)
+
+This week most of what I accomplished was Research into how game maker language works. I looked into it scalability as well and how it handles complex and Abstract data types. I also found a couple of YouTube channels that offer very helpful tutorials, namely Shaun Spalding and FriendlyCosmonaut. While I don't have a whole lot of code to show for this week, I can say that I have a much more confident understanding of how gml works and how best to use it.
+
+
+## 2. What challenges or difficulties did you face? If you solved them, how? If not, what have you learned so far? Have you sought help or other resources?
+
+The biggest challenge or difficulty that I faced this week was determining a direction to go in. While it is true that I had deliverable two to fall back on so that I had a way to push forward, figuring out some of the smaller details on how to tackle specific issues took a little bit more figuring out. Because of the research that I have now done and the resources I have gathered, I feel more prepared to tackle these smaller steps that I've laid out for myself.
+
+
+## 3. What do you plan to accomplish in the following week? (Please list at least 3 concrete "S.M.A.R.T." goals, along with estimated number of hours to complete, or day to complete)
+
+Next week I'm going to actually properly dive into the major issues that I'm going to handle from deliverable two.  Tell prepare myself for that I issue-ified most of the major points on that document so that I would have more specific and measurable targets. I feel like they're perfectly obtainable and reasonable in the time that I have, and I've broken them up in such a way that I expect each of them to take one full work week to complete. There is going to be some wiggle room in that because I still have such an abstract far out and you of how to accomplish these goals but I feel I've broken them down to manageable components.
+
+
+## 4. What resources will you use to accomplish your goals for the upcoming week? (please list out the resources)
+
+The resources I will use to accomplish my goals will mostly be the yo-yo documentation for gml, but I will also be making heavy use of the YouTube channels I referenced for question 1.
```

Comment:
Leaving this here for documentation; R0 Feedback was commented on a previous PR.